Sources/SwiftCliCore/InputValidator.swift

```diff
@@ -15,6 +15,20 @@ public class InputValidator {
         return isItValid
     }
     
+
+    public func FindWidestRow(patternarray: [[String]]) -> Int {
```

Comment:
I'd encourage you to stick to Swift style conventions, which prefers `camelCase` for both functions and parameter names, so `findWidestRow` and `patternArray`.